### PR TITLE
NetworkIOMeter: convert packets to per second

### PR DESCRIPTION
**Pull request by eworm-de**
We should normalize the packets as well...

### CODE REVIEW

**Review by BenBE — APPROVED** [review]

File: NetworkIOMeter.c [hunk 1]
```diff
@@ -77,8 +78,8 @@ static void NetworkIOMeter_updateValues(Meter* this) {
 
       if (data.bytesTransmitted > cached_txb_total) {
          diff = data.bytesTransmitted - cached_txb_total;
-         diff /= ONE_K; /* Meter_humanUnit() expects unit in kilo */
          diff = (1000 * diff) / passedTimeInMs; /* convert to per second */
+         diff /= ONE_K; /* Meter_humanUnit() expects unit in kilo */
```
Comment (BenBE) [inline NetworkIOMeter.c:82]: Same as above:

```suggestion
         diff /= passedTimeInMs; /* convert to B/ms */
         diff = (1000 * diff) / ONE_K; /* convert to KiB/s */
```

**Review by BenBE — CHANGES_REQUESTED** [review]
I meant it that way around. I hope I didn't screw up with the intermediate units.

Please squash/fixup those changes into your second commit.

File: NetworkIOMeter.c [hunk 2]
```diff
@@ -59,8 +59,8 @@ static void NetworkIOMeter_updateValues(Meter* this) {
 
       if (data.bytesReceived > cached_rxb_total) {
          diff = data.bytesReceived - cached_rxb_total;
-         diff /= ONE_K; /* Meter_humanUnit() expects unit in kilo */
          diff = (1000 * diff) / passedTimeInMs; /* convert to per second */
+         diff /= ONE_K; /* Meter_humanUnit() expects unit in kilo */
```
[the remark this answers is not in the document]
Comment (eworm-de) [reply NetworkIOMeter.c]: But dividing at the beginning to get bytes per **millisecond** is the worst thing for precision, no?

[the remark this answers is not in the document]
Comment (Explorer09) [reply NetworkIOMeter.c]: I don't get this either. If rounding to bytes per second is possible, there is no sense to round to bytes per _millisecond_ in the first place.

**Pull request merged**